build.sbt

```diff
@@ -25,7 +28,8 @@ lazy val `kyo-settings` = Seq(
     fork               := true,
     scalaVersion       := scala3Version,
     crossScalaVersions := List(scala3Version),
-    scalacOptions ++= compilerOptions,
+    scalacOptions     ++= scalacOptionTokens(compilerOptions).value,
+    Test / scalacOptions --= scalacOptionToken(ScalacOptions.languageStrictEquality).value,
```

Comment:
Doesn't this disable strict equality for tests? I'd prefer to keep it on since it's easy to make mistakes and compare kyo computations directly to pure values.